### ISSUE TITLE
1.1.30 desktop files/icons not installed

### ISSUE THREAD

**Issue by barjac** [sadko4u/lsp-plugins#193]
Using our existing packaging that works fine for 1.1.29, all the build appears error free, but no desktop files or icons are installed during the build.
I notice a big change in the makefile since 1.1.29 relating to the desktop files/icons, has something been missed?
http://mtf.duckdns.org/pub/linux/barjac/distrib/cauldron/x86_64/log/lsp-plugins-1.1.30-1.mga9_bcj.src.rpm/build.x86_64.0.20210504213020.log

**Comment by sadko4u** [comment]
Hello!

Since I've got many claims from GNOME users that LSP Plugins are flooding menu with icons, I decided to delegate the responsibility for them to the distribution maintainers.

That's why I've made an additional makefile target to install the icons which is not working by default now:
```
make install_xdg
```

But any package builder can add it to the build chain to have the desired effect.

Ready to discuss the stuff.
